src/delorean.js

```diff
@@ -160,6 +160,7 @@
           // you can change it using `DeLorean.Flux.define('Promise', AnotherPromise)`
           return new DeLorean.Promise(function (resolve, reject) {
             store.listener.once('change', resolve);
+            store.listener.once('rollback', reject);
```

Comment:
so this is a problem that has bugged me for awhile... nothing is cleaning up these listeners. They use `once`, so they self cleanup as long as the event is emitted. This arrangement is acceptable when only a `change` event is bound. The caveat being that a store must fire a `change` event in response to every dispatched action. 
However, a store obviously cannot fire both a `change` and `rollback` event, on every `dispatch` as that makes no sense. So, the result will be a memory leak where listeners are continually bound, but never removed. 
The bad news is that we cannot merge this PR until this issue is resolved. The good news is that we have to solve this issue, IMO, it is currently biggest flaw with delorean.